src/Views/Components/AdminUI/Button/style.module.scss

```diff
@@ -0,0 +1,76 @@
+.button {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    font-size: 1rem;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.5;
+    letter-spacing: normal;
+    border-radius: var(--givewp-rounded-4);
+    cursor: pointer;
+    border: 1px solid transparent;
+
+    > svg {
+        transform: translate(8px, 1px);
+    }
+}
+
+.primary {
+    color: var(--givewp-shades-white);
+    background: var(--givewp-blue-500);
+
+    &:hover {
+        background-color: var(--givewp-blue-400);
+    }
+
+    &:active {
+        background-color: var(--givewp-blue-600);
+    }
+}
+
+.secondary {
+    background: var(--givewp-shades-white);
+    color: var(--givewp-blue-500);
+    border-color: var(--givewp-blue-500);
+
+    &:hover {
+        color: var(--givewp-blue-400);
+        border-color: var(--givewp-blue-400);
+    }
+
+    &:active {
+        color: var(--givewp-blue-600);
+        border-color: var(--givewp-blue-600);
+    }
+}
+
+.danger {
+    background-color: var(--givewp-red-500);
+    color: var(--givewp-shades-white);
+    border-color: var(--givewp-red-500);
+
+    &:hover {
+        background-color: var(--givewp-red-400);
+    }
+
+    &:active {
+        background-color: var(--givewp-red-600);
+    }
+}
+
+.small {
+    padding: var(--givewp-spacing-2) var(--givewp-spacing-5);
+}
+
+.large {
+    padding: var(--givewp-spacing-4) var(--givewp-spacing-20);
+}
+
+.button:disabled {
+    background-color: var(--givewp-grey-300);
+    color: var(--givewp-shades-white);
+    border-color: var(--givewp-grey-300);
+}
```

Comment:
As this is a state of the button element, it would make more sense if staying closer to it:
```scss
.button {
    ...

    &:disabled {
        background-color: var(--givewp-grey-300);
        color: var(--givewp-shades-white);
        border-color: var(--givewp-grey-300);
    }
}

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6726/commits/8786d71f09125c0225e79bff0ac95dc922ef68b3

Reply:
I think my suggestion is slightly different from your implementation. I meant to keep it close as you have the `.button` class on line 1, so it makes sense just to move the `:disabled` statement to be appended to the class on the top.

Reply:
resolved: https://github.com/impress-org/givewp/pull/6726/commits/3ae7d14428acfabba13d64c6f2f3638ec2a21d9e